pandas/core/indexes/base.py

```diff
@@ -98,9 +100,11 @@
 )
 
 if TYPE_CHECKING:
-    from pandas import Series
+    from pandas import RangeIndex, Series
 
 
+str_ = str
```

Comment:
i think elsewhere we use "str_t" for this

Reply:
we use both `str_t` and `str_type` so why no have another?

only jesting. see https://github.com/pandas-dev/pandas/pull/32365#issuecomment-592987245

will change to str_t for now.